util/execdetails/execdetails.go

```diff
@@ -52,3 +54,58 @@ func (d ExecDetails) String() string {
 	}
 	return strings.Join(parts, " ")
 }
+
+// RuntimeStats collects executors's execution info.
+type RuntimeStats map[string]*RuntimeStat
+
+// RuntimeStat collects one executor's execution info.
+type RuntimeStat struct {
+	// executor's Next() called times.
+	loop int32
+	// executor consume time.
+	consume int64
+	// executor return row count.
+	rows int64
+}
+
+// NewRuntimeStats creates new executor collector.
+func NewRuntimeStats() RuntimeStats {
+	return RuntimeStats(make(map[string]*RuntimeStat))
+}
+
+// GetRuntimeStat gets execStat for a executor.
+func (e RuntimeStats) GetRuntimeStat(planID string) *RuntimeStat {
+	if e == nil {
+		return nil
+	}
+	runtimecStat, exists := e[planID]
+	if !exists {
+		runtimecStat = &RuntimeStat{}
+		e[planID] = runtimecStat
+	}
+	return runtimecStat
+}
+
+func (e RuntimeStats) String() string {
```

Comment:
this method can be removed? BTW, the result is not stable, because the it is composed by a `range` operation on a `map`.

Reply:
it's current no use and can be remove